azure-pipelines.yml

```diff
@@ -137,7 +137,7 @@ stages:
           displayName: Build / Publish
           condition: succeeded()
 
-        - ${{ if eq(variables._RunAsPublic, False) }}:
+        - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
```

Comment:
Since there's the exact same if a few lines below, I think it would make sense to move the CG step to be under the other if?

Reply:
I also had this idea but there is actually this other line outside of the if in between:
```
- template: /eng/test.yaml
```